src/main/java/network/brightspots/rcv/SecurityConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I'm super annoyed that this diff doesn't seem to be working properly... this file was already exists in `hotfix/1.3.2` because it was created by #764 which has now been merged in. I've tried changing the base back and forth in GitHub. This isn't really reviewable as-is. Any idea how to fix this problem?

Reply:
This is the issue caused by squash commits + chained PRs that we noted a while back.

My workflow is to cherry-pick only the relevant commits and force-push to the branch, e.g.:

`Base` < `Feature1` [commit1, commit2] < `Feature2` [commit3, commit4]
When `Feature1` merges into `Base`:
```
git checkout Feature2
git reset --hard Base
git cherrypick commit3..commit4
git push --force
```

Reply:
Is it worth adding a section to the Wiki about this, or will you just keep it on your end as domain knowledge, since it seems to be specific to your workflow (unless I'm misunderstanding)?

Reply:
Can do: https://github.com/BrightSpots/rcv/wiki/Notes-on-Development

Maybe worth adding the IntelliJ setup you use to this wiki too (I think it's on the README now?)

Reply:
Follow-up on this: could this have been avoided if, in your local branch and after the first squash was merged into the main branch, you pulled back from the main branch (with the new squash merge) into your local branch and reconciled changes? I believe this is what I've historically done in these situations, but there might be some subtle difference I'm missing.

Reply:
Yes, that's what I used to do, but reconciling differences is manual and error-prone, especially if there are dozens. Bugs have slipped in before caused by this (e.g. #670), so I don't recommend it.

This method is less error-prone.

Reply:
Ok cool. It's generally pretty painless with IntelliJ -- I'll mention that method in the Wiki as well.

Reply:
Filed #771 for updating Wiki with IntelliJ info.